### ISSUE TITLE
Multiuser fails to load authlibs

### ISSUE THREAD

**Issue by juztas** [opensciencegrid/xrootd-multiuser#21]
From https://github.com/opensciencegrid/xrootd-multiuser/issues/14 I create a separate issue for multiuser and 5.x release (just not to mix issue/solution). 
Before:
```
ofs.authlib libXrdAccSciTokens.so libXrdMacaroons.so
```
After (good):
```
ofs.authlib ++ libXrdAccSciTokens.so
ofs.authlib ++ libXrdMacaroons.so
```
It works if I am not using multiuser plugin (read, dir listing, etc) - but does not work with multiuser. So this seems an issue on multiuser plugin:
```
Plugin loaded Multiuser v5.1.1 from fslib libXrdMultiuser-5.so
------ Initializing the multi-user plugin.
=====> multiuser.umask 0022
=====> ofs.authlib ++ libXrdAccSciTokens.so
=====> ofs.authlib ++ libXrdMacaroons.so
210323 14:54:23 3815349 multiuser_Config: Failed to load ++-5 ++-5: cannot open shared object file: No such file or directory
210323 14:54:23 3815349 multiuser_Initialize: Encountered a runtime failure: Failed to configure multi-user plugin.
210323 14:54:23 3815349 XrootdConfig: Unable to load file system via libXrdMultiuser.so
210323 14:54:23 3815349 XrootdConfig: Unable to load file system wrapper from libXrdMultiuser.so
------ xroot protocol initialization failed.
```
Reply from @abh3 https://github.com/opensciencegrid/xrootd-multiuser/issues/14#issuecomment-805297173
```
Hi Justas,

Ah, I was not aware that multi-user reparses the configuration and is
sensitive to those directives. Sigh, another thing that should not have
been done. Now, I will need to find out why it was done that way in the
first place.

Andy
```

**Comment by juztas** [comment]
@abh3 do you have a timeline for this fix? Thanks

**Comment by abh3** [comment]
Hi Justas,

Derek is now in control of the timeline. The patches for him to 
reimplement multiuser are already in git master.

Andy

**Comment by djw8605** [comment]
Fixed in #22
